com.github.lhy0403.Qv2ray.json

```diff
@@ -0,0 +1,93 @@
+{
+    "app-id": "com.github.lhy0403.Qv2ray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "qv2ray",
+    "rename-desktop-file": "qv2ray.desktop",
+    "rename-icon": "qv2ray",
+    "rename-appdata-file": "qv2ray.metainfo.xml",
+    "cleanup": [
+        "/include",
+        "/lib/debug",
+        "*.la",
+        "*.a"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=~/.qv2ray",
+        "--filesystem=~/.config/qv2ray",
```

Comment:
Any reason why the app doesn't use XDG dirs?

Reply:
How can I use XDG dirs for ```~/.qv2ray```?

Reply:
By using xdg dirs, the application will store it's settings in a sandboxed(other flatpak apps can't access it) data dir under `~/.var/app/$app-id/`. So you don't need any of those two filesystem in such case. Unless it's used for something else

Reply:
Thanks. But we need that to store some user settings.

Reply:
Well, you can store user settings by using https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html. I just don't see why you want those settings to be in `$HOME/.config/qv2ray` or `$HOME/.qv2ray` instead of just `$XDG_CONFIG_HOME/qv2ray/`
By using that, the settings will be stored under `$HOME/.var/app/com.github.lhy0403.Qv2ray/config/qv2ray` under Flatpak and in `$HOME/.config/qv2ray` otherwise.

Reply:
@ymshenyu  ^^^